libs/lensfun/mod-coord.cpp

```diff
@@ -43,11 +43,12 @@ lfLensCalibDistortion rescale_polynomial_coefficients (const lfLensCalibDistorti
     const float hugin_scale_in_millimeters =
         hypot (36.0, 24.0) / lcd.CalibAttr.CropFactor / hypot (lcd.CalibAttr.AspectRatio, 1) / 2.0;
     const float hugin_scaling = real_focal / hugin_scale_in_millimeters;
+    float d = 1;
```

Comment:
I’d prefer setting `scaling_d` in every case and leave `d` a const. After all, `d` has a special meaning, and is not present in the other models.